RecoMTD/DetLayers/src/MTDDetRing.cc

```diff
@@ -58,20 +60,22 @@ pair<bool, TrajectoryStateOnSurface> MTDDetRing::compatible(const TrajectoryStat
 vector<GeometricSearchDet::DetWithState> MTDDetRing::compatibleDets(const TrajectoryStateOnSurface& startingState,
                                                                     const Propagator& prop,
                                                                     const MeasurementEstimator& est) const {
-  const std::string metname = "MTD|RecoMTD|RecoMTDDetLayers|MTDDetRing";
-
-  LogTrace(metname) << "MTDDetRing::compatibleDets, Surface at Z: " << surface().position().z()
-                    << " R1: " << specificSurface().innerRadius() << " R2: " << specificSurface().outerRadius()
-                    << " TS at Z,R: " << startingState.globalPosition().z() << ","
-                    << startingState.globalPosition().perp() << "     DetRing pos." << position();
+#ifdef EDM_ML_DEBUG
+  LogTrace("MTDDetLayers") << "MTDDetRing::compatibleDets, Surface at Z: " << surface().position().z()
+                           << " R1: " << specificSurface().innerRadius() << " R2: " << specificSurface().outerRadius()
+                           << " TS at Z,R: " << startingState.globalPosition().z() << ","
+                           << startingState.globalPosition().perp() << "     DetRing pos." << position();
+#endif
 
   vector<DetWithState> result;
 
   // Propagate and check that the result is within bounds
   pair<bool, TrajectoryStateOnSurface> compat = compatible(startingState, prop, est);
   if (!compat.first) {
-    LogTrace(metname) << "    MTDDetRing::compatibleDets: not compatible"
-                      << "    (should not have been selected!)";
+#ifdef EDM_ML_DEBUG
+    LogTrace("MTDDetLayers") << "    MTDDetRing::compatibleDets: not compatible"
+                             << "    (should not have been selected!)";
+#endif
```

Comment:
having EDM_ML_DEBUG around LogTrace looks completely redundant.
this looks like an incomplete cleanup (I see a few more in other places of this PR).
Or did I miss something?